### DISCUSSION TITLE
2009/01/30/%E6%89%93%E9%80%A0%E5%85%8D%E8%B4%B9%E4%B8%AA%E4%BA%BA%E8%AE%A1%E7%AE%97%E6%9C%BA%E5%AE%89%E5%85%A8%E7%B3%BB%E7%BB%9F

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#251] [Announcements]
# 2009/01/30/%E6%89%93%E9%80%A0%E5%85%8D%E8%B4%B9%E4%B8%AA%E4%BA%BA%E8%AE%A1%E7%AE%97%E6%9C%BA%E5%AE%89%E5%85%A8%E7%B3%BB%E7%BB%9F

最近一阵子，我试用了一些市面上常见的免费计算机安全相关软件，找到了两个较为适合个人计算机上使用的杀毒软件。我家用的电脑正在使用这两款杀毒软件，现在推荐给对杀毒软件不是太了解的计算机用户。

https://blog.qizhen.xyz/2009/01/30/%E6%89%93%E9%80%A0%E5%85%8D%E8%B4%B9%E4%B8%AA%E4%BA%BA%E8%AE%A1%E7%AE%97%E6%9C%BA%E5%AE%89%E5%85%A8%E7%B3%BB%E7%BB%9F

**Comment by ruanqizhen** [comment]
之前博客下的留言：

wuximing
2013年05月11日 10:50 编辑
最后一段的“尤其”，改为“但是”表转折才更妥当。

回复
俊
2009年02月26日 09:08 编辑
我在自己的机器上这样部署安全环境的：Symantec Endpoint Protection 11，赛门铁克的企业版杀毒软件，客户端，免费升级病毒库，具备病毒和间谍软件的防护以及网络防火墙功能（可以详细列编条目，很有效，基于端口和基于应用都有）windows 清理助手（ARSWP），一款绿色软件，免费，有一个行之有效的团队或称为联盟在支持，无需常住内存，需要的时候偶尔开启升级库文件，然后扫描即可，很多中毒病入膏肓的机器，杀毒软件进程都kill了，用arswp扫描一遍，就想打扫卫生一样，恢复正常。
